orchard_sinsemilla.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+import sys; assert sys.version_info[0] >= 3, "Python 3 required."
+
+import math
+
+import orchard_iso_pallas
+
+from pyblake2 import blake2b, blake2s
+from orchard_pallas import Fp, p, q, PALLAS_B, Point
+from orchard_iso_pallas import PALLAS_ISO_B, PALLAS_ISO_A
+from sapling_utils import i2beosp, cldiv, beos2ip, i2leosp, lebs2ip
+from binascii import hexlify
+from bitstring import BitArray
```

Comment:
AFAICT this is an external module, so we need to update the requirements in the README.

Reply:
It looks like this dependency can just be removed in favour of using strings of '0' and '1'.